dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -37,6 +37,11 @@
     /** The stores before every method call. */
     protected final IdentityHashMap<Block, TransferInput<A, S>> stores;
 
+    /** The results for each node. */
```

Comment:
Please expand this documentation and state what the different keys and values to the maps are and what they are caching.